src/magic_folder/testing/web.py

```diff
@@ -413,9 +413,14 @@ def render_GET(self, request):
         capability = None
         for arg, value in uri.query:
             if arg == u"uri":
+                if capability is not None:
+                    raise Exception("Can't sepecify uri twice in query args.")
```

Comment:
This isn't what Tahoe will do, I don't think -- and anyway isn't used (according to codecov) so just delete.

Reply:
I'd be find moving them to a separate PR so that it doesn't block this one.

Reply:
well, it's un-tested code and arguably this stuff _should_ represent "whatever Tahoe does" (even if you think Tahoe is wrong) .. right?

Reply:
> should represent "whatever Tahoe does"

I would rather say, it should ensure that we are using the API correctly. That is very often done by making the test double behave the same as the real thing. But I think making the test double more pedantic is also a reasonable thing to do.

Reply:
Fine, if you think that's required it at least needs a test.

Reply:
I've removed these changes from the PR.